web/rootfs/etc/cont-init.d/10-config

```diff
@@ -85,6 +85,18 @@ if [[ ! -f /config/config.js ]]; then
         /config/config.js
     fi
 
+    if [[ $ENABLE_XMPP_WEBSOCKET -eq 1 ]]; then
+        if [[ ! -z "$PUBLIC_URL" ]]; then
+            sed -i \
+              -e "s#// websocket:.*#websocket: '${PUBLIC_URL/http/ws}/xmpp-websocket',#" \
+            /config/config.js
+        else
```

Comment:
Exit with an error otherwise please. Thsi will only lead to trouble as it doesn't work on mobile (and never will, because config.js is evaluated on a separate JS VM for security reasons).

Reply:
nice to know, thanks.